File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.java
class Solution {
    public String solution(int[] food) {
        String answer = "";
        String s = "";
        for(int i=1; i<food.length; i++){
            int num = food[i]/2;
            
            for(int j=0; j<num; j++){
                s += i;
            }
        }
        
        StringBuffer sb = new StringBuffer(s);
        String sr = sb.reverse().toString();
        return s + 0 + sr;
    }
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.kt
class Solution {
    fun solution(food: IntArray): String {
        var s = ""
        for(i in 1 until food.size){
            var num = food[i]/2
            for(j in 0 until num){
                s += i
            }
        }
        return s + 0 + s.reversed()
    }
}